smoke/helpers.go

```diff
@@ -9,13 +10,34 @@ import (
 	. "github.com/onsi/gomega/gexec"
 )
 
+var DEFAULT_TIMEOUT = 30 * time.Second
+
+func SetBackend(appName string) {
+	config := GetConfig()
+	if config.Backend == "diego" {
+		EnableDiego(appName)
+	} else if config.Backend == "dea" {
+		DisableDiego(appName)
+	}
+}
+
 func EnableDiego(appName string) {
+	guid := GetAppGuid(appName)
+	Eventually(cf.Cf("curl", "/v2/apps/"+guid, "-X", "PUT", "-d", `{"diego": true}`), DEFAULT_TIMEOUT).Should(Exit(0))
+}
+
+func DisableDiego(appName string) {
```

Comment:
Yes, it seems reasonable that they are consistent - we were starting from the assumption that we would make `EnableDiego` unexported too. Given we won't change that (see line note below) then it's fine to leave this one as-is.